src/main/java/com/salesforce/functions/jvm/sdk/bulk/builder/JobBuilder.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+package com.salesforce.functions.jvm.sdk.bulk.builder;
+
+import com.salesforce.functions.jvm.sdk.bulk.*;
+import javax.annotation.Nonnull;
+
+/**
+ * Builder for {@link Job}.
+ *
+ * @see BulkApi#newJobBuilder(String, Operation)
+ */
+public interface JobBuilder {
```

Comment:
Is this written as an interface for discussion purposes only?  It will be a concrete type when it comes time to implement, right?